internal/worker/server.go

```diff
@@ -133,7 +133,11 @@ func (s *Server) EnqueueManifestJobByID(job *ManifestJobByID, parent uuid.UUID,
 }
 
 func (s *Server) enqueue(jobType string, job interface{}, dependencies []uuid.UUID, channel string) (uuid.UUID, error) {
-	prometheus.EnqueueJobMetrics(jobType)
+	tenant := channel
+	if len(channel) > 4 {
+		tenant = channel[4:]
```

Comment:
How come we chop off the prefix?

Reply:
Interesting. I'm not sure why we have the prefix, so this might be a stupid question, but could we just pass the whole thing as the tenant ID and decode it (if necessary) when we consume it. In case we ever have other channels with other prefixes, I mean? If we do want to chop off the prefix, we should probably check for the string "org-" explicit, and if that is not set, error out  (or ignore it, whatever is appropriate). That way changes in the future won't take us by surprise.

cc @ondrejbudai